packages/@expo/cli/src/utils/workflow.ts

```diff
@@ -0,0 +1,62 @@
+import { AndroidConfig, IOSConfig } from 'expo/config-plugins';
```

Comment:
This file is copied from eas-updates (but modified to account for this use-case)